Automattic/wp-calypso
# Remove Help Center ETK

Follow up for https://github.com/Automattic/jetpack/pull/38093.

## Proposed Changes

Removes the Help Center from ETK.

## Testing Instructions

1. cd into `apps/editing-toolkit`.
2. run `yarn dev --sync`. 
3. make sure the Help Center still works.
